user/mods/ServerValueModifier/Presets/The Escape.json

```diff
@@ -499,22 +499,22 @@
   "CSM": {
     "EnableCases": true,
     "EnableSecureCases": false,
-    "CustomPocket": true,
+    "CustomPocket": false,
     "Pockets": {
       "DefaultPocket": true,
       "SpecGKeychain": false,
-      "SpecSimpleWallet": true,
-      "SpecWZWallet": true,
+      "SpecSimpleWallet": false,
+      "SpecWZWallet": false,
       "SpecKeycardHolder": false,
       "SpecKeytool": false,
       "SpecInjectorCase": false,
       "SpecSlots": 3,
       "FourthH": 1,
       "FourthV": 1,
-      "ThirdH": 1,
-      "ThirdV": 2,
-      "SecondH": 1,
-      "SecondV": 2,
+      "ThirdH": 2,
+      "ThirdV": 1,
+      "SecondH": 2,
+      "SecondV": 1,

```

Comment:
Even if it's disabled, this is not the correct values here

Reply:
as I disabled it, I didn't look at this numbers

Reply:
fixed